admin_ui/src/components/ArrayWidget.vue

```diff
@@ -13,7 +13,12 @@
                 </a>
             </li>
             <li>
-                <a href="#" v-on:click.prevent="addArrayElement">
+                <a
+                    class="add"
+                    href="#"
+                    v-on:click.prevent="addArrayElement"
+                    v-if="enableAddButton"
+                >
```

Comment:
I don't know if this helps, but you can write whole file like this because with that we can see every single file in array widget (I don't know if you have any other idea but it doesn't work at the moment)
```vue
<template>
    <div>
        <ul class="array_items">
            <li :key="index" v-for="(value, index) in internalArray">
                <input
                    :type="inputType"
                    :value="value"
                    id="choice"
                    v-on:change="updateArray($event, index)"
                />
                <a
                    href="#"
                    v-on:click.prevent="showMedia(index)"
                    :value="value"
                    v-if="isMediaColumn"
                    ><font-awesome-icon icon="eye" title="View"
                /></a>
                <a href="#" v-on:click.prevent="removeArrayElement(index)">
                    <font-awesome-icon icon="times" />
                </a>
            </li>
            <li>
                <a
                    class="add"
                    href="#"
                    v-on:click.prevent="addArrayElement"
                    v-if="enableAddButton"
                >
                    <font-awesome-icon icon="plus" />{{ $t("Add") }}
                </a>
            </li>
        </ul>
        <MediaViewer
            v-if="showMediaViewer"
            :mediaViewerConfig="mediaViewerConfig"
            @close="showMediaViewer = false"
        />
    </div>
</template>

<script lang="ts">
import MediaViewer from "./MediaViewer.vue"
import { MediaViewerConfig } from "@/interfaces"

export default {
    props: {
        array: {
            type: Array,
            default: () => []
        },
        inputType: {
            type: String,
            default: "text"
        },
        title: {
            type: String,
            default: ""
        },
        enableAddButton: {
            type: Boolean,
            default: true
        }
    },
    components: {
        MediaViewer
    },
    data() {
        return {
            internalArray: [],
            showMediaViewer: false,
            mediaViewerConfig: null as MediaViewerConfig,
            index: 0
        }
    },
    computed: {
        schema() {
            return this.$store.state.schema
        },
        currentTableName() {
            return this.$store.state.currentTableName
        },
        isMediaColumn() {
            return this.schema.media_columns.includes(
                this.getFieldName(this.title).toLowerCase()
            )
        }
    },
    methods: {
        updateArray($event, index) {
            this.$set(this.internalArray, index, $event.target.value)
            this.$emit("updateArray", this.internalArray)
        },
        addArrayElement() {
            this.internalArray = [...this.internalArray, ""]
            this.$emit("updateArray", this.internalArray)
        },
        removeArrayElement(index) {
            this.$delete(this.internalArray, index)
            this.$emit("updateArray", this.internalArray)
        },
        getFieldName(name: string) {
            return name.toLowerCase().split(" ").join("_")
        },
        showMedia(index) {
            const mediaViewerConfig: MediaViewerConfig = {
                fileKey: this.internalArray[index],
                columnName: this.getFieldName(this.title),
                tableName: this.currentTableName
            }
            this.mediaViewerConfig = mediaViewerConfig
            this.showMediaViewer = true
        }
    },
    watch: {
        array() {
            this.internalArray = [...this.array]
        }
    },
    mounted() {
        this.internalArray = [...this.array]
    }
}
</script>
```

Reply:
You're right - the `view` button doesn't work for the array column at the moment.

What you've suggested here looks good.

Reply:
Thanks. You can also remove the `static` folder as it is no longer used.